apps/omg_db/lib/omg_db/rocks_db.ex

```diff
@@ -74,25 +74,11 @@ defmodule OMG.DB.RocksDB do
     GenServer.call(server_name, {:utxo, utxo_pos})
   end
 
-  def exit_infos(server_name \\ @server_name) do
-    _ = Logger.info("Reading exits' info, this might take a while. Allowing #{inspect(@one_minute)} ms")
```

Comment:
It seems this log wasn't moved to the new version of that function? With the `@one_minute` timeout.

Reply:
yeah, I started with logging this in the model layer. However, just realized that everyone with this log is calling `get_all_by_type/2` with 1 minute timeout. So I move the log to there instead.